Bithumb/Bithumb/CoinDetail/Components/OrderBookListView/OrderBookListViewCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  OrderBookListViewCell.swift
+//  Bithumb
+//
+//  Created by 이영우 on 2022/02/07.
+//
+
+import Foundation
+
+import SnapKit
+import Then
+
+final class OrderBookListViewCell: UITableViewCell {
+
+  // MARK: Properties
+
+  private let bidQuantitylabel: PaddingLabel
+  private let askQuantityLabel: PaddingLabel
+  private let priceLabel: PaddingLabel
+  private let priceChangedRatioLabel: PaddingLabel
+  private let priceStackView: UIStackView
+
+
+  // MARK: Initializer
+
+  override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+    let padding = UIEdgeInsets(top: 6, left: 6, bottom: 6, right: 6)
+    self.priceChangedRatioLabel = PaddingLabel(padding: padding)
+    self.bidQuantitylabel = PaddingLabel(padding: padding)
+    self.askQuantityLabel = PaddingLabel(padding: padding)
+    self.priceLabel = PaddingLabel(padding: padding)
+    self.priceStackView = UIStackView()
+
+    super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+    self.attribute()
+    self.layout()
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  private func attribute() {
+    [self.askQuantityLabel, self.bidQuantitylabel, self.priceLabel, self.priceChangedRatioLabel].forEach {
+      $0.textAlignment = .left
+      $0.textColor = .black
+      $0.font = .systemFont(ofSize: 12)
+    }
+  }
+
+  private func layout() {
+    [self.bidQuantitylabel, self.askQuantityLabel, self.priceStackView].forEach {
+      self.contentView.addSubview($0)
+    }
+
+    [self.priceLabel, self.priceChangedRatioLabel].forEach {
+      self.priceStackView.addArrangedSubview($0)
+    }
+
+    self.bidQuantitylabel.snp.makeConstraints {
+      $0.leading.equalToSuperview()
+      $0.top.bottom.equalToSuperview().inset(2)
+      $0.width.equalToSuperview().multipliedBy(0.3)
+    }
+
+    self.askQuantityLabel.snp.makeConstraints {
+      $0.leading.equalTo(self.priceStackView.snp.trailing).offset(2)
+      $0.trailing.equalToSuperview()
+      $0.top.bottom.equalToSuperview().inset(2)
+      $0.width.equalToSuperview().multipliedBy(0.3)
+    }
+
+    self.priceStackView.snp.makeConstraints {
+      $0.top.bottom.equalToSuperview().inset(2)
+      $0.leading.equalTo(self.bidQuantitylabel.snp.trailing).offset(2)
+    }
+
+    self.priceChangedRatioLabel.setContentHuggingPriority(.required, for: .horizontal)
+    self.priceChangedRatioLabel.setContentCompressionResistancePriority(.required, for: .horizontal)
+  }
+
+  override func prepareForReuse() {
```

Comment:
OrderBook(ask, bid)의 경우에 따라 Cell 마다 달라지는 속성이 많아 prepareForReuse 를 통해서 재설정했는데, 공식 문서에서 성능상의 이슈를 언급하고 있네요. 아래 내용에 대해서 같이 이야기 나눠 보면 좋을 것 같아요!

> To avoid potential performance issues, you should only reset attributes of the cell that are not related to content, for example, alpha, editing, and selection state.

Reply:
오 좋네요! 
공식문서에서 언급하는 content는 어떤 의미일까요? 🤔

Reply:
음 Cell 내부에 들어갈 View들의 Text나 image 등을 초기화시키는 동작들을 의미하는 것 같아요!

Cell의 Attribute만 reset 시키는데 사용하라고 하는 것을 보니, 저희는 해당 안될 것 같네요. 제이는 어떻게 생각하시나요?

Reply:
네! 저도 공식문서에서 언급한 `not related to content~` 라는 부분이 저희 상황이라고 생각해서 없어도 되지 않을까 생각해요 :)

Reply:
네넵 그렇다면 반영하도록 할게요~ 3a612c0